### PR TITLE
Docs: Interactive Diagram Setting stored params best practices

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] - Docs: Setting storedParams best practices when dependent on previous values

### CODE REVIEW

File: docs/interactive-diagram-best-practices.md [hunk 1]
```diff
@@ -57,3 +57,44 @@ To run the above example, run the following command and upload the `output/index
 EXAMPLE=toggle-panels npm run compile-interactive-example
 ```
 
+## Setting Stored Parameters
+
+### Mental Modal
```
Comment (JulianGoh5) [inline docs/interactive-diagram-best-practices.md:62]: Should this be "mental model"?

**Review by JulianGoh5 — APPROVED** [review]
Found the description and diagrams very thorough, nice work!

File: docs/interactive-diagram-best-practices.md (hunk 1 above)
Comment (michaellisitsa) [reply docs/interactive-diagram-best-practices.md:62]: Yes. Definitely model

Comment (michaellisitsa) [reply docs/interactive-diagram-best-practices.md:62]: Will include spelling in next PR https://github.com/ClearCalcs/custom-diagram-boilerplate/pull/43 which touches this same file, as have to resolve conflicts anyway

Comment (michaellisitsa) [reply docs/interactive-diagram-best-practices.md:62]: Fixed in 4201c9e20236d8af00ecbfc5b84ec43f8f3ec89f

**Pull request merged**